admin/modules/config/mod_tools.php

```diff
@@ -865,7 +865,8 @@ function checkAction(id)
 
 			if($mybb->get_input('threadprefix', MyBB::INPUT_INT) >= 0)
 			{
-				$thread_options['threadprefix'] = $mybb->get_input('threadprefix', MyBB::INPUT_INT);
+				$thread_options['threadprefix'] = $mybb
+                    ->get_input('threadprefix', MyBB::INPUT_INT);
```

Comment:
I meant this line when speaking about the syntax error, but I may have noticed wrong and it possibly doesn't throw anything. But It should be reverted regardless - no need to separate it into a new line.